lib/npm.js

```diff
@@ -113,19 +113,20 @@
             'See the README.md or bin/npm-cli.js for example usage.'
```

Comment:
Line 106: `var registryRefer` is defined by never used
Line 95: `var littleGuys` is a reference to incorrect spelling, but `lib/config/cmd-list.js` already has an `affordances` section; shouldn't we be using that?

Reply:
Yes, registryRefer should be removed, since we're not sending the referer header any more.

Istr there's a difference between "affordances" and "littleGuys", but I don't recall it offhand.  Will dig in and sort that out.

Reply:
Nope, no difference.  Just apparently something that got overlooked 4 years ago when it stopped being relevant.